### DISCUSSION TITLE
Live: section of config

### DISCUSSION THREAD

**Discussion by MrCaspan** [blakeblackshear/frigate#15069] [Ask A Question] [answered]
In the config refrence you have the following

```
live:
  # Optional: Set the name of the stream configured in go2rtc
  # that should be used for live view in frigate WebUI. (default: name of camera)
  # NOTE: In most cases this should be set at the camera level only.
  stream_name: camera_name
  # Optional: Set the height of the jsmpeg stream. (default: 720)
  # This must be less than or equal to the height of the detect stream. Lower resolutions
  # reduce bandwidth required for viewing the jsmpeg stream. Width is computed to match known aspect ratio.
  height: 720
  # Optional: Set the encode quality of the jsmpeg stream (default: shown below)
  # 1 is the highest quality, and 31 is the lowest. Lower quality feeds utilize less CPU resources.
  quality: 8
```

I am still not sure about the `height` setting. I have go2rtc doing all my streams and I am using my 4K stream for live views but this says dont set it above your detect stream. Why are we setting the height of the live stream (4K) using the detect height? It does say sset the height of the jsmpeg stream which makes me think this setting only applies if you dont have live views set to your high res stream? Very confusing and hoping you can help explain it!

**Comment by NickM-27** [comment, marked as answer]
It specifically says the height of the jsmpeg stream, it doesn't apply to go2rtc
